test/integration/suites/sync-authorized-entries/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+  spire-server:
+    image: spire-server:latest-local
+    volumes:
+      - ./conf/server:/opt/spire/conf/server
+    command: ["-config", "/opt/spire/conf/server/server.conf"]
+  spire-agent:
+    image: spire-agent:latest-local
+    volumes:
+      - ./conf/agent:/opt/spire/conf/agent
+    command: ["-config", "/opt/spire/conf/agent/agent.conf"]
+  bad-spire-agent:
+    image: spire-agent:latest-local
+    volumes:
+      - ./conf/bad-agent:/opt/spire/conf/agent
+    command: ["-config", "/opt/spire/conf/agent/agent.conf"]
```

Comment:
why it is required?

Reply:
Its not, just some leftovers from the test I started from. I thought I cleaned those up, but turns out i didn't